### ISSUE TITLE
operative_config.gin not found

### ISSUE THREAD

**Issue by umer11792** [google-research/text-to-text-transfer-transformer#740]
**Error**
Checked on default t5 colab notebook and the issue persists.

When I fine-tune a model i.e. small/base. It shows following lines:
- INFO:root:system_path_file_exists:gs://t5-data/pretrained_models/base/operative_config.gin
- ERROR:root:Path not found: gs://t5-data/pretrained_models/base/operative_config.gin

Fine-tuning works fine. It causes an error while evaluating the model with following messages:
INFO:root:system_path_file_exists:gs://t5_header/models/base/operative_config.gin
ERROR:root:Path not found: gs://t5_header/models/base/operative_config.gin
/usr/local/lib/python3.7/dist-packages/t5/seqio/preprocessors.py:65: UserWarning: Creating resources inside a function passed to Dataset.map() is not supported. Create each resource outside the function, and capture it inside the function to use it.
  _tokenize, num_parallel_calls=tf.data.experimental.AUTOTUNE)
INFO:absl:Skipping packing/padding for 'flag1' since sequence length is None.
INFO:absl:Skipping packing/padding for 'flag2' since sequence length is None.
INFO:absl:Skipping packing/padding for 'flag3' since sequence length is None.
INFO:absl:Skipping packing/padding for 'flag4' since sequence length is None.

**Comment by adarob** [comment]
I don't have access to `gs://t5_header` -- have you checked that the operative config is there?

**Comment by umer11792** [comment]
Yes. That's my bucket. I have checked the config file and it exists. I checked this error against the default t5 notebook and it has the same error on default settings and only change was I mentioned my own GCS. Notice there are two messages:

1. system_path_file_exists: gs://t5-data/pretrained_models/base/operative_config.gin
2. Path not found:                gs://t5-data/pretrained_models/base/operative_config.gin

**Comment by antonio-mastropaolo** [comment]
I had the same issue a couple of weeks ago. I didn't experience any problem during the training nor the fine-tuning, and when I went to GCS, to check the gin, it was there. 
A quick workaround (colab) to suppress those warnings is to load the gin file in colab and then point to that one. 
Like this:

```
PATH_GIN_FILE = '/content/operative_config.gin'
import gin

with gin.unlock_config():
    gin.parse_config_file(PATH_GIN_FILE)
    
    #RUN FINE-TUNING
    TRAIN_STEPS = 300000
    model.train("task", steps=TRAIN_STEPS)

```

Hopefully this help :)

**Comment by apoorvumang** [comment]
Hi antonio

I'm trying to train the model on my task from scratch. I tried what you suggested. The operative_config.gin I'm using is thru
`! gsutil cp gs://t5-data/pretrained_models/small/operative_config.gin .`` Then I used the following snippet for loading config and training

```
PATH_GIN_FILE = '/content/operative_config.gin'
import gin
with gin.unlock_config():
  gin.parse_config_file(PATH_GIN_FILE)

  TRAIN_STEPS = 300000
  model.train(
    mixture_or_task_name="wikidata5m_tail_prediction_unique_alias",
    steps = TRAIN_STEPS
  )
```

But I'm getting the following error
```
ModuleNotFoundError: No module named 't5.data.sentencepiece_vocabulary'
  In file "/content/operative_config.gin", line 2
    import t5.data.sentencepiece_vocabulary
```

Any help would be appreciated
